metis/bin/metis_client

```diff
@@ -501,6 +506,7 @@ EOT
     attr_reader :options
 
     def initialize(shell)
+      @headless = ENV['METIS_headless'] == 'true' ? true : false
```

Comment:
First, thanks so much for putting in a PR -- this is awesome!

super-nit: is the convention for environment variables to be in all caps? i.e. `METIS_HEADLESS`

Reply:
haha yeah that's a good point.... it should be ALL_CAPS. Will fix in the next version!